analysis/report_templates/default.html

```diff
@@ -232,7 +232,25 @@ <h5 class="center-align">Mann-Whitney U test</h4>
                 </li>
             </ul>
 
-
+            {% if coverage_report %}
+            <ul class="collapsible">
+                <li>
+                    <div class="collapsible-header">
+                        Coverage reports for each fuzzer on this benchmark
+                    </div>
+                    <div class="collapsible-body">
+                        <div class="row">
+                            {% for fuzzer in benchmark.fuzzers %}
+                            <div class="col">
+                                <a class="waves-effect waves-light btn" href="https://storage.googleapis.com/{{ experiment_filestore_name }}/{{ experiment.name }}/reports/coverage/{{ benchmark.name }}/{{ fuzzer }}/index.html">{{ fuzzer }}</a>
```

Comment:
Is there a reports sub-dir ? i dont think so.

Reply:
I created that sub-dir and put all the html there